classification/view_all_classified_ics.m

```diff
@@ -0,0 +1,58 @@
+function view_all_classified_ics(class_file,ica_mat,label_option)
+% View ICs on top of cell map, where color of outline indicates
+%   cell (green) or not (cell)
+%   
+%   class_file = classification text file
+%   ica_mat = ica mat file
+%   label_option = 'y' if you want IC number labels; 'n' if not
+%
+% 2015 02 19 Fori Wang
+
+figure;
+
+% load ICs
+load(ica_mat);
+
+% Pull out classification data
+fid = fopen(class_file);
+results = textscan(fid,'%s %s %s %s');
```

Comment:
After changing the code, you need to re-push to the branch to update the PR.

(I'm bringing the postdoc candidate from Clark.)